components/secret-mgt/org.wso2.carbon.identity.secret.mgt.core/src/main/java/org/wso2/carbon/identity/secret/mgt/core/SecretManagerImpl.java

```diff
@@ -289,7 +380,7 @@ private SecretDAO getSecretDAO() throws SecretManagementException {
     private void validateSecretManagerEnabled() throws SecretManagementServerException {
 
         if (!SecretManagerComponentDataHolder.getInstance().isSecretManagementEnabled()) {
-            throw handleServerException(ERROR_CODE_SECRET_MANAGER_NOT_ENABLED, DB_TABLE_NAME);
+            throw handleServerException(ERROR_CODE_SECRET_MANAGER_NOT_ENABLED, DB_TABLE_SECRET);
```

Comment:
let's not throw the table name to the caller. This should be internal detail.

Reply:
resolved